src/indra_cogex/sources/pubmed/__init__.py

```diff
@@ -89,7 +89,7 @@ def get_relations(self):
                             "MESH",
                             mesh_id,
                             "annotated_with",
-                            {"is_major_topic": True if major_topic == "1" else False},
```

Comment:
🚀

Reply:
Sorry @cthoyt, had to extend the expression again after reading the[nedo4j admin manual](https://neo4j.com/docs/operations-manual/current/tools/neo4j-admin/neo4j-admin-import/#import-tool-header-format-properties) again carefully:

> Boolean values are true if they match exactly the text true. All other values are false.

Fixed in https://github.com/bgyori/indra_cogex/pull/55/commits/7b40c913efcc8308229beb69fedb0e5f6a8e138c